ops/cli.py

```diff
@@ -172,6 +172,7 @@ def run_liquibase_migration(db_username, db_password):
     changelog_dir = f"{pwd}/wfl/database"
     command = ' '.join(['docker run --rm --net=host',
                         f'-v {changelog_dir}:/liquibase/changelog liquibase/liquibase',
+                        '/liquibase/liquibase ',
```

Comment:
```suggestion
                        '/liquibase/liquibase',
```
I don't think the extra space matters either way.